src/SamplesApp/SamplesApp.Wasm/Properties/launchSettings.json

```diff
@@ -3,8 +3,8 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:49266/",
-      "sslPort": 44309
+      "applicationUrl": "http://localhost:57069/",
+      "sslPort": 44325
     }
```

Comment:
Was this intended?

Reply:
No, but 2022 changes is every time now... I'll remove it.